lms/validation/_launch_params.py

```diff
@@ -53,10 +50,7 @@ def lti_user(self):
 
         :rtype: LTIUser
         """
-        try:
-            kwargs = parser.parse(self, self.context["request"], locations=["form"])
-        except HTTPUnprocessableEntity as err:
-            raise ValidationError(err.json) from err
+        kwargs = self.parse(locations=["form"])
```

Comment:
Same thing here: `self.parse()` already raises `ValidationError` not `HTTPUnprocessableEntity`, so it's not necessary to wrap the exception here